packages/mobile-app/metro.config.js

```diff
@@ -56,41 +40,63 @@ const emptyModules = new Set([
 
-  // Optionally, chain to the standard Metro resolver.
-  return context.resolveRequest(context, moduleName, platform);
-};
+  config.resolver = {
+    ...resolver,
+    nodeModulesPaths: [
+      path.resolve(projectRoot, "node_modules"),
+      path.resolve(monorepoRoot, "node_modules"),
+    ],
+    disableHierarchicalLookup: true,
+    resolveRequest: (context, moduleName, platform) => {
+      if (shims[moduleName]) {
+        return shims[moduleName];
+      }
 
-// Node.js built-in modules that are unused.
-// These will only be replaced with empty modules if they can't be found in the project's node_modules.
-// If the project includes a module with the same name, it will be used instead.
-// Modules with slashes in the name need to go in emptyModules instead.
-config.resolver.extraNodeModules = {
-  ...config.resolver.extraNodeModules,
-  assert: config.resolver.emptyModulePath,
-  child_process: config.resolver.emptyModulePath,
-  crypto: config.resolver.emptyModulePath,
-  events: config.resolver.emptyModulePath,
-  fs: config.resolver.emptyModulePath,
-  http: config.resolver.emptyModulePath,
-  net: config.resolver.emptyModulePath,
-  os: config.resolver.emptyModulePath,
-  stream: config.resolver.emptyModulePath,
-  tls: config.resolver.emptyModulePath,
-  url: config.resolver.emptyModulePath,
-  util: config.resolver.emptyModulePath,
-  worker_threads: config.resolver.emptyModulePath,
-  v8: config.resolver.emptyModulePath,
-};
+      if (emptyModules.has(moduleName)) {
+        return {
+          type: "empty",
+        };
+      }
+
+      // Optionally, chain to the standard Metro resolver.
+      return context.resolveRequest(context, moduleName, platform);
+    },
+    // Node.js built-in modules that are unused.
+    // These will only be replaced with empty modules if they can't be found in the project's node_modules.
+    // If the project includes a module with the same name, it will be used instead.
+    // Modules with slashes in the name need to go in emptyModules instead.
+    extraNodeModules: {
+      ...resolver.extraNodeModules,
+      assert: resolver.emptyModulePath,
+      child_process: resolver.emptyModulePath,
+      crypto: resolver.emptyModulePath,
+      events: resolver.emptyModulePath,
+      fs: resolver.emptyModulePath,
+      http: resolver.emptyModulePath,
+      net: resolver.emptyModulePath,
+      os: resolver.emptyModulePath,
+      stream: resolver.emptyModulePath,
+      tls: resolver.emptyModulePath,
+      url: resolver.emptyModulePath,
+      util: resolver.emptyModulePath,
+      worker_threads: resolver.emptyModulePath,
+      v8: resolver.emptyModulePath,
+    },
+    assetExts: resolver.assetExts.filter((ext) => ext !== "svg"),
+    sourceExts: [...resolver.sourceExts, "svg"],
+  };
+
+  config.transformer = {
+    ...transformer,
+    unstable_allowRequireContext: true,
+    babelTransformerPath: require.resolve("react-native-svg-transformer/expo"),
```

Comment:
Added this one option, the rest of `config.transformer` should be the same as it was before.